elearning-backend/routes/api.php

```diff
@@ -22,13 +23,20 @@
 
 Route::middleware('auth:sanctum')->group(function () {
     // Authenticated Routes
+    Route::get('/user', [UserProfileController::class, 'index']);
     Route::post('/user', [UserProfileController::class, 'index']);
-    Route::post('/update-user', [UserController::class, 'update']);
+    Route::patch('/update-user', [UserController::class, 'update']);
     Route::post('/logout', [SessionsController::class, 'logout']);
 
     // Admin Routes
     Route::get('/admin/get-users', [AdminUsersController::class, 'index']);
     Route::get('/admin/get-user/{id}', [AdminUsersController::class, 'getUser']);
     Route::patch('/admin/update-user', [AdminUsersController::class, 'update']);
     Route::delete('/admin/delete-user/{id}', [AdminUsersController::class, 'delete']);
+    
+    Route::get('/admin/get-categories', [AdminCategoriesController::class, 'index']);
+    Route::get('/admin/get-category/{id}', [AdminCategoriesController::class, 'getCategory']);
+    Route::post('/admin/store-category', [AdminCategoriesController::class, 'store']);
+    Route::put('/admin/update-category/{id}', [AdminCategoriesController::class, 'update']);
+    Route::delete('/admin/delete-category/{id}', [AdminCategoriesController::class, 'delete']);
```

Comment:
```suggestion
    Route::get('/admin/categories', [AdminCategoriesController::class, 'index']);
    Route::get('/admin/categories/{id}', [AdminCategoriesController::class, 'getCategory']);
    Route::post('/admin/categories', [AdminCategoriesController::class, 'store']);
    Route::put('/admin/categories/{id}', [AdminCategoriesController::class, 'update']);
    Route::delete('/admin/categories/{id}', [AdminCategoriesController::class, 'delete']);
```
Pls follow convention by laravel, read more here
https://laravel.com/docs/9.x/controllers#actions-handled-by-resource-controller

Note: it will not conflict even though they have same routes because they have different method.

Reply:
Routes methods and api calls for this module are fixed. I will update all other routes and api calls on separate bugfix branches.